buildkitd/buildkitd.go

```diff
@@ -355,10 +355,7 @@ func Start(ctx context.Context, console conslogging.ConsoleLogger, image, contai
 			})
 		}
 
-		bkURL, err := url.Parse(settings.BuildkitAddress)
-		if err != nil {
-			panic("Buildkit address was not a URL when attempting to start buildkit")
-		}
+		bkURL, _ := url.Parse(settings.BuildkitAddress) // error is already handled in addRequiredOpts
```

Comment:
I would personally prefer to leave the error handling in. While this error may _currently_ be handled in `addRequiredOpts`, that may not always be the case - and if someone removes it from `addRequiredOpts` they probably won't know to update it here. This is a classic case of "comments are just lies waiting to happen."

Reply:
makes sense, will update, I mostly didn't like the panic but I suppose I can just adjust that instead (I'm guessing we used panic because it was unexpected given the previous err handling, but I could be wrong).